### ISSUE TITLE
Couldn't start client Kotlin Language Server on Windows 10

### ISSUE THREAD

**Issue by adiRandom** [fwcd/KotlinLanguageServer#57]
Hey there!
I've installed the extension from the market on a windows 10 pc and after setting up the env variable for the JDk and kotlin everything went smoothly. But when I went ahead and install the extension on my main PC I got this notification ,,Couldn't start client Kotlin Language Server" even after setting up the env variables under PATH.
The only log that I get is the following:
`[Error - 7:41:05 PM] Starting client failed
Launching server using command c:\Users\Adrian\.vscode\extensions\fwcd.kotlin-0.1.8\build\install\kotlin-language-server\bin\kotlin-language-server.bat failed.
` 

I've already reinstalled the extension and reset the env vars. When I went to check the path from the log to see if there is a problem with that script I realized that there is no build directory which is obviously the source of the error.

Can anyone help me out please?

**Comment by adiRandom** [comment]
Alright. Just did that. So, to sum up everything seems to work now, but in case this is a somewhat common bug, here's what I did:
-Cloned the git repo
-Ran `gradlew installDist` inside the repo folder

After that I've got this error:
 ```
> Task :fixFilePermissions FAILED

FAILURE: Build failed with an exception.

 What went wrong:
Execution failed for task ':fixFilePermissions'.
> A problem occurred starting process 'command 'echo "Performed build on Windows (thus not requiring additional file permissions)"''
```

I went ahead and unchecked the read only permission from windows folder properties, but that didn't helped. The build folder got created and I could copy it in the extension directory which seemed to have solved the problem.

Thanks for the help!
